src/libwalletqt/Wallet.h

```diff
@@ -175,6 +181,7 @@ class Wallet : public QObject
     mutable QTime   m_daemonBlockChainTargetHeightTime;
     mutable quint64 m_daemonBlockChainTargetHeight;
     int     m_daemonBlockChainTargetHeightTtl;
+    QObject * m_walletManager;
```

Comment:
Thanks for feedback. Removed. 
OK. I got the dreaded "QObject: Cannot create children for a parent that is in a different thread." when i used Wallet as parent. Changed to creating PendingTransaction without parent and warning is gone. Since PT is deleted in disposeTransaction() i hope that is an OK solution?